cli/common.go

```diff
@@ -95,11 +95,16 @@ func maybeBlockWithFeedback(ctx context.Context, bctx blockWithFeedbackCtx) {
 			os.Exit(1)
 		}
 
+		if controller.IsUnitsAlreadyUpToDate(taskObject.Error) {
+			newLogger.Info(ctx, "Not updating group '%s': (%s)", bctx.Request.Group, taskObject.Error.Error())
```

Comment:
To match the other outputs. https://github.com/giantswarm/inago/blob/no-err-if-update-not-needed/cli/common.go#L105